### ISSUE TITLE
Mod Request: FastBench for Fabric

### ISSUE THREAD

**Issue by Brottweiler** [ReperakPro/fabric-optimization-mods#13]
**Requirements for a mod to be added:**
- [x] The mod must be open-source. 
- [x] The mod cannot sacrifice vanilla behavior for performance.
- [x] The mod must be actively maintained.

**Mod name:** FastBench for Fabric

**Type of mod (e.g. rendering, lighting, extra):** tick optimization (I think)

**Repository (e.g. GitHub, GitLab, Gitea):** https://github.com/Tfarcenim/FabricFastBench

**Developer/organization:** tfarecnim

**Additional notes:**
Curseforge link: https://www.curseforge.com/minecraft/mc-mods/fastbench-for-fabric

**Comment by DragonEggBedrockBreaking** [comment]
Should be closed due to inactivity.

**Comment by Brottweiler** [comment]
The latest version works for the latest Minecraft version as of today.

**Comment by DragonEggBedrockBreaking** [comment]
> The latest version works for the latest Minecraft version as of today.

So does fastfurnace. That has nothing to do with inactivity.

**Comment by DragonEggBedrockBreaking** [comment]
I just PR'd to fastbench, so it may be updated soon.

**Comment by mr-pi12** [comment]
There's a mod called fastbench minus replacement which is still being updated and according to the dev, is less invasive because it doesn't replace the registry and uses mixins instead

**Comment by Brottweiler** [comment]
I actually don't use FastBench or the equivalent for Furnaces. I found "Recipe Cache" that seems to work better. FastBench made crafting output invisible on some servers for me.

**Comment by mr-pi12** [comment]
Yeah, I opened an issue this morning, but haven't tested it thoroughly myself. I also got the same invisible crafting bug, check my issue.
